app/src/main/java/org/cru/godtools/ui/dashboard/tools/ToolsCategoryDataModel.kt

```diff
@@ -13,17 +13,25 @@ import org.cru.godtools.model.Tool
 import org.keynote.godtools.android.db.Contract.ToolTable
 import org.keynote.godtools.android.db.GodToolsDao
 
+private const val ATTR_SELECTED_CATEGORY = "selectedCategory"
+
 @HiltViewModel
-class ToolsCategoryDataModel @Inject constructor(dao: GodToolsDao) : ViewModel() {
+class ToolsCategoryDataModel @Inject constructor(dao: GodToolsDao, private val savedState: SavedStateHandle) :
+    ViewModel() {
+    var selectedCategory: String?
+        get() = savedState.get(ATTR_SELECTED_CATEGORY)
+        set(value) { savedState.set(ATTR_SELECTED_CATEGORY, value) }
```

Comment:
you probably don't need this property, you should be able to use the selectedCategoryLiveData directly